public/script.js

```diff
@@ -3325,9 +3325,18 @@ export async function Generate(type, { automatic_trigger, force_name2, quiet_pro
     // Make quiet prompt available for WIAN
     setExtensionPrompt('QUIET_PROMPT', quiet_prompt || '', extension_prompt_types.IN_PROMPT, 0, true);
     const chatForWI = coreChat.map(x => `${x.name}: ${x.mes}`).reverse();
-    const { worldInfoString, worldInfoBefore, worldInfoAfter, worldInfoExamples, worldInfoDepth } = await getWorldInfoPrompt(chatForWI, this_max_context, dryRun);
+    var { worldInfoString, worldInfoBefore, worldInfoAfter, worldInfoExamples, worldInfoDepth } = await getWorldInfoPrompt(chatForWI, this_max_context, dryRun);
     setExtensionPrompt('QUIET_PROMPT', '', extension_prompt_types.IN_PROMPT, 0, true);
 
+    {
+        let regexType = regex_placement.USER_INPUT;
```

Comment:
That's not very transparent. Why USER INPUT type?

Reply:
```ts
const regex_placement: {
    MD_DISPLAY: number;
    USER_INPUT: number;
    AI_OUTPUT: number;
    SLASH_COMMAND: number;
}
```
There is no other more appropriate enumeration type...

Reply:
It will be a good idea to apply regex entry-by-entry in the `checkWorldInfo` function before returning them to the prompt builder.

Reply:
fixed at [`e65d43a` (#2193)](https://github.com/SillyTavern/SillyTavern/pull/2193/commits/e65d43ad1aaf1451522dfa06a878d1d24f61bcd0) !